src/OpenTracing/Span.php

```diff
@@ -15,10 +15,13 @@ public function operationName();
     public function context();
 
     /**
+     * Finishes the span and locks it so it becomes immutable. As an implementor, make
```

Comment:
I believe this is a necessary recommendation, does it make sense?
@beberlei @felixfbecker